application/src/main/java/com/sanction/lightning/resources/FacebookResource.java

```diff
@@ -64,4 +67,127 @@ public Response getUser(@Auth Key key, @QueryParam("username") String username)
 
     return Response.ok(facebookUser).build();
   }
+
+  /**
+   * Fetches all the photos of a specific user.
+   * @param key The authentication key for the requesting application.
+   * @param username The username of the StormUser to get FacebookUser information for.
+   * @return Array of photos
+   */
+  @GET
+  @Path("/photos")
+  public Response getPhotos(@Auth Key key, @QueryParam("username") String username)
+          throws IOException, MalformedURLException {
```

Comment:
I don't like these `throws`. Can we catch them and return an appropriate response status?